nilearn/reporting/tests/test_reporting.py

```diff
@@ -13,7 +13,7 @@
 )
 
 # Avoid making pyflakes unhappy
-_set_mpl_backend
+_set_mpl_backend  # noqa: B018
```

Comment:
Hmm... Need to investigate this but I think this may just be there to pacify linters that complain about an unused import, but we need to import it to set up the plotting backend.

Probably best to then silence the unused import linting error